metriq_gym/process.py

```diff
@@ -0,0 +1,156 @@
+"""Benchmark processing and calculation utilities."""
+import json
+import math
+import statistics
+
+from scipy.stats import binom
+
+from qiskit_ibm_runtime import QiskitRuntimeService
+from qiskit.providers import Job
+
+from metriq_gym.bench import BenchJobResult, BenchProvider
+
+def get_job(result: BenchJobResult) -> Job:
+    if result.provider == BenchProvider.IBMQ:
+        return QiskitRuntimeService().job(result.id)
+
+    raise ValueError(f"Cannot find provider {job.provider}.")
+
+
+def get_job_result(job: Job, partial_result: BenchJobResult):
+    result = job.result()
+    partial_result.counts = result.get_counts()
+    partial_result.interval = result.time_taken
+
+    return partial_result
+
+
+def poll_job_results(jobs_file: str) -> [BenchJobResult]:
```

Comment:
`[BenchJobResult]` should be `list[BenchJobResult]`

Reply:
Done.